en/components/date_range_picker.md

```diff
@@ -118,11 +118,11 @@ Both the [`IgxDateRangeStartComponent`]({environment:angularApiUrl}/classes/igxd
 
 #### User Experience
 
-In the default configuration, with a single read-only input, the calendar can be opened by clicking anywhere in the input, including the calendar icon. When there are two separate inputs for start and end date since they are editable by default, the calendar can only be opened from the calendar icon.
+In a default configuration with a single read-only input, the calendar can be opened by clicking anywhere in the input, including the calendar icon. When there are two separate inputs for start and end date, the calendar can only be opened from the calendar icon, since both inputs are editable by default.
```

Comment:
```suggestion
In a default configuration, with a single read-only input, the calendar can be opened by clicking anywhere in the input, including the calendar icon. When there are two separate inputs for start and end date, the calendar can only be opened from the calendar icon, since both inputs are editable by default.
```